.github/workflows/ci.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
       shell: bash
       run: |
         set -xmveuo pipefail
-        git submodule update --init || true
+        git submodule update --init --remote || true
```

Comment:
This is IMHO not the way, submodules should be used. They are powerful, because they refer to a known to work commit in the subrepository.
And the CI should show, that this specific combination of revisions work.

Reply:
Maybe we want another kind of relation between those repositories, but then we should stop using submodules.

Reply:
@mdellweg I just noticed this comment now.

In the pulplift CI PR, I actually updated the user instructions with "--remote":
https://github.com/pulp/pulplift/pull/95/files#diff-04c6e90faac2675aa89e2176d2eec7d8R28

I am open to other ideas, I just think this is the best way of handling this for now. Particular in terms of maintenance. Devs should just run that command from time to time.

In the long run, a bot would be great, but we are using Travis, not GHA. And I thought bots are less feasible on Travis.

If other repos start to break us, we will at least find out in our CI, and be able to quickly adjust the CI scripts and instructions.